lasio/las.py

```diff
@@ -704,6 +704,43 @@ def set_data_from_df(self, df, **kwargs):
                 str(name) for name in df.columns.values
             ]
         self.set_data(df_values, **kwargs)
+    
+    def stack_curves(self, curve_list=None, stub=None, sort_curves=True):
+        """Stack multi-channel curve data to a numpy 2D ndarray. Provide a 
+        list of curves or a stub name (prefix shared by all curves that will
+        be stacked). If `stub` and `curve_list` are both supplied, `stub`
+        is used.
+        
+        Keyword Arguments:
+            stub (str): Supply the first several characters of the channel
```

Comment:
I like the stub argument. Looks great.

Reply:
Cool, glad you like it. I've maintained the functionality with my latest commit b4678cb3b4b34ff0dc19db4637b81306e7a70915 but the api changed a little